tests/test_otel.py

```diff
@@ -25,6 +25,8 @@
         interval {{ interval or "1ms" }};
         batch_size 3;
         batch_count 3;
+
+        {{ exporter_opts }}
```

Comment:
Probably, you mean here this:
```
    otel_exporter {
      {{ exporter_opts }}
    }
```

Reply:
No, I want other directives to have proper test defaults. I guess, we can call it "exporter_extra".